pyqtorch/modules/hamevo.py

```diff
@@ -343,3 +343,20 @@ def forward(self, H: torch.Tensor, t: torch.Tensor, state: torch.Tensor) -> torc
         """
         ham_evo_instance = self.get_hamevo_instance(H, t)
         return ham_evo_instance.forward(state)
+
+
+class HamEvoSparseAnalog(Module):
```

Comment:
@dominikandreasseitz why is this called `Analog`? Is it because you support only global qubit support?